src/core/src/Rotation.cpp

```diff
@@ -517,6 +517,28 @@ namespace iDynTree
         return map;
     }
 
+    Matrix3x3 Rotation::RPYRightTrivializedDerivativeRateOfChange(const double /*roll*/, const double pitch, const double yaw, const double /*rollDot*/, const double pitchDot, const double yawDot)
+    {
+        Matrix3x3 map;
+
+        double sp = std::sin(pitch);
+        double cp = std::cos(pitch);
+        double sy = std::sin(yaw);
+        double cy = std::cos(yaw);
+
+        map(0, 0) = -sp * cy * pitchDot - cp * sy * yawDot;
```

Comment:
Is there a reference for these computations?

Reply:
Actually no, I wrote the math on my notebook

Reply:
Do you think it may make sense to add few lines about the approach you used?

Reply:
In the end, I've just evaluated the component-wise derivative.

Reply:
Yes sorry, I should have looked at the derivative methods just above.